tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/SlidingWindowTaggedMetricRegistry.java

```diff
@@ -21,18 +21,18 @@
 
 public final class SlidingWindowTaggedMetricRegistry extends AbstractTaggedMetricRegistry {
 
-    private static final TaggedMetricRegistry DEFAULT = new SlidingWindowTaggedMetricRegistry(1, TimeUnit.MINUTES);
-
     public SlidingWindowTaggedMetricRegistry(int window, TimeUnit windowUnit) {
         super(() -> new SlidingTimeWindowArrayReservoir(window, windowUnit));
     }
 
     /**
      * Get the global default {@link TaggedMetricRegistry}.
+     * @deprecated inline this
      */
     @SuppressWarnings("unused") // public API
+    @Deprecated
     public static TaggedMetricRegistry getDefault() {
-        return SlidingWindowTaggedMetricRegistry.DEFAULT;
+        return SharedTaggedMetricRegistries.getDefault();
```

Comment:
I'm somewhat inclined to just delete this method completely and cut a `0.12.0` breaking release as I don't believe anyone actually uses this.